packages/jsii-calc/lib/submodule/refers-to-parent/.jsiirc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "targets": {
+    "java": {
+      "package": "software.amazon.jsii.tests.calculator.refers_to_parent"
```

Comment:
Is the plan to also support java and .net in this PR or as a separate PR?

Reply:
I was thinking probably subsequent PR, not settled yet.